src/rpp/stack_trace.cpp

```diff
@@ -1094,9 +1104,9 @@ namespace rpp
         return count;
     }
 
-    RPPAPI std::vector<uint64_t> get_callstack(size_t maxDepth, size_t entriesToSkip) noexcept
+    RPPAPI std::vector<uint64_t> _get_callstack(size_t maxDepth, size_t entriesToSkip, HANDLE thread = nullptr) noexcept
```

Comment:
if you add a helper function, it shifts the stack trace depth by 1 actually, it was very carefully aligned before
You could instead do:
```cpp
RPPAPI std::vector<uint64_t> get_callstack(size_t maxDepth, size_t entriesToSkip, uint64_t threadId = 0) noexcept;
```
Which would be cross platform compatible
And delegate the thread opening to ThreadContext